src/crypto/OutgoingRoomKeyRequestManager.js

```diff
@@ -200,15 +220,22 @@ export default class OutgoingRoomKeyRequestManager {
                         // do.)
                         this._sendOutgoingRoomKeyRequestCancellation(
                             updatedReq,
+                            andResend,
                         ).catch((e) => {
                             console.error(
                                 "Error sending room key request cancellation;"
                                 + " will retry later.", e,
                             );
                             this._startTimer();
-                        }).done();
+                        }).then(() => {
+                            // The request has transitioned from
+                            // CANCELLATION_PENDING_AND_WILL_RESEND to UNSENT. We
```

Comment:
has it? are you sure?

Reply:
oops, I've removed a check I had for `andResend`. Will re-add